packages/plugins/experimental/plugin-script/src/types/script.ts

```diff
@@ -5,16 +5,23 @@
 import { ref, S, TypedObject } from '@dxos/echo-schema';
 import { TextType } from '@dxos/plugin-markdown/types';
 
+// TODO(burdon): Which is the outer type? I.e., in the navbar?
```

Comment:
these types were not intended to be hierarchical, the script reference for functions is an optional reference denoting origin rather than ownership